doc/setting_up_vcpkg.md

```diff
@@ -0,0 +1,94 @@
+# Building applications using vcpkg for C SDK
+
+This document describes how to setup vcpkg to build applications using Microsoft Azure IoT device SDK for C. It demonstrates building sample projects in Windows using the Visual Studio project files included in the SDK. It also demonstrates building standalone applications using cmake project files in Linux and Mac.
+
+### Setup C SDK vcpkg for Windows development environment
+
+- Open a command prompt and run the following commands:
+
+```Shell
+
+# Clone vcpkg
+git clone https://github.com/Microsoft/vcpkg.git vcpkg_new
+
+# Bootstrap and install Visual Studio integration
+pushd vcpkg_new
+.\bootstrap-vcpkg.bat
+vcpkg integrate install
+
+# Install azure-iot-sdk-c package
```

Comment:
By MD rules you need a black line after #, or at least that's what Visual Studio Code tells me.